config.json

```diff
@@ -74,6 +75,15 @@
       "bitwise operations",
       "performance optimizations"
     ]
+  }, {
+    "difficulty": 3,
+    "slug": "pangram",
+    "topics": [
+      "filtering",
```

Comment:
I'm not sure that this really covers filtering. I think this really refers to whittling down a collection to a smaller collection.